src/provider/rpc.ts

```diff
@@ -194,7 +194,7 @@ export class RpcProvider implements ProviderInterface {
     return this.fetchEndpoint('starknet_getTransactionReceipt', { transaction_hash: txHash });
   }
 
-  public async getClass(classHash: RPC.Felt): Promise<RPC.ContractClass> {
+  public async getClassByHash(classHash: RPC.Felt): Promise<RPC.ContractClass> {
```

Comment:
I would recommend creating an alias function the same as we did with getBlock(), and leaving the original name as is.

Reply:
Just for RPC or for the Provider Interface?

Reply:
I thought only RPC as the endpoint is get_class.
I am not sure how many users we have that will be impacted but in general, I think it is good to do if we can soft transition. where we create alias depercate and than remove in couple of versions in the future.
like:
v1:
methodA()

v2:
@deprecated - use methodB() 
methodA()
...
methodB()

vN:
methodB()